practices/graded-browser-support.md

```diff
@@ -1,67 +1,90 @@
+* Grade-A are capable or very popular browser versions. These browsers will receive the full experience, with all styling and behaviour. Bugs are addressed with high priority.
+* Grade-C are incapable and/or uncommon browser versions which are known to break if receiving full styling and behaviour. These browsers will receive a core experience, with only minimum CSS and JavaScript. Bugs are addressed with high priority.
+* Grade-X browsers exist in the grey area between Grade-A and Grade-C. We serve them the same code as Grade-A, so they will receive the full experience, but we do not test our products against these browser versions. We expect our sites to work in these browsers. Over time, analysis of errors and usage patterns may cause some browsers in Grade-X to be deliberately changed to either Grade-A or Grade-C, to allow for better support.
 
-| Tables                      | Grade-A support                 | Grade-C support    |
-| --------------------------- |:-------------------------------:| ------------------:|
-| Chrome                      | latest stable, latest stable -1 | < 29               |
-| Edge                        | latest stable, latest stable -1 | n/a                |
-| Firefox                     | latest stable, latest stable -1 | < 29               |
-| IE                          | 11                              | < 11               |
-| Opera                       | latest stable                   | < 16               |
-| Safari iOS                  | latest stable, latest stable -1 | < 7                |
-| Safari MacOS                | latest stable, latest stable -1 | < 6.1              |
-| Android browser (Chromium)  | Latest stable                   | < 4.4              |
+## Graded browser support list
 
-### Notes
+This is the current list of browser versions and their corresponding grades:
 
-- We have no explicit Grade-C support for Edge.
-- We have no Grade-X support for IE, as we officially only support IE11, with anything below that being Grade-C. However, due to browser technical limitations we can only detect IE10+, so IE10 is, for now, included in our CTM media query. 
+| Tables                    | Grade-A support                 | Grade-C support |
+| ------------------------- |:-------------------------------:| ---------------:|
+| Chrome                    | latest stable, latest stable -1 | < 29            |
+| Edge                      | latest stable, latest stable -1 | none            |
+| Firefox                   | latest stable, latest stable -1 | < 29            |
+| IE                        | 11                              | < 11            |
+| Opera                     | latest stable                   | < 16            |
+| Safari iOS                | latest stable, latest stable -1 | < 7             |
+| Safari MacOS              | latest stable, latest stable -1 | < 6.1           |
+| Android browser (Webview) | Latest stable                   | < 4.4           |
```

Comment:
Not sure why this changed from Chromium to Webview? You have the browser bundled with the OS (Chrome) and the browser engine that powers webviews (a chromeless browser used by hybrid apps), which is now powered by Chromium.

Reply:
Well, the Android browser has always been Webview, but Webview only started using Chromium as of Android 4.4. Older versions of Webview didn't use Chromium, and we specifically added that browser to the list as we made versions older than 4.4 (so, the non-Chromium based ones) Grade-C.

Reply:
Yep, but "android browser" and "webview" are different things, which is not what this table row implies... Chrome releases across platforms have parity, so I think that's encompassed by the "Chrome" entry at the top of the table. So this column row is pretty much redundant these days, apart from calling out old webkit-based webviews.

So these days I think this row is about webview, not the browser -- especially as that is what the version number in the Grade-C column is referring to. How about we change the text from "Android browser (Webview)" to just "Android Webview"? Also the version number in the final column is not a browser version, but an OS version, so [maybe change that to "Android  < 4.4.3"](https://developer.chrome.com/multidevice/webview/overview#webview_faq)

And yes, I did read your PR notes 😁

Reply:
I agree, the "Android browser" description is not right. I've replaced it with "Android Webview" now. I've also found the version, which is 30.0.0.0, so I've made it "< 30 (Android 4.4)".

Ref: https://developer.chrome.com/multidevice/webview/overview